packages/capsule/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/capsule",
-  "version": "2.1.0",
+  "version": "3.0.0-alpha.1",
```

```suggestion
  "version": "2.2.0-alpha.1",
```
As I dont see a breaking change we can just bump with a minor version